modules/virt-template-fields-web.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+The following table outlines the fields when creating a new custom template in the {VirtProductName} web console. Not all fields are editable.
```

Comment:
I think it would be better only to include editable fields and to remove the last sentence. If you try to document all fields, you will run into maintenance problems because UXD moves fields around all the time.

Reply:
Asciidoc issue. It might be problematic to begin this module with text. As far as I know, a header1 is the only permitted way to begin a module.